server/app/views/questiontypes/PhoneQuestionRenderer.java

```diff
@@ -78,26 +56,16 @@ protected DivTag renderInputTags(
                     phoneQuestion.getPhoneNumberPath(), ImmutableSet.of()))
             .setScreenReaderText(applicantQuestion.getQuestionTextForScreenReader())
             .addReferenceClass(ReferenceClasses.PHONE_NUMBER)
```

Comment:
question (non-blocking): do we still need this reference class?

Reply:
At least for now. A lot of tests started to fall apart when I removed/changed it.